### ISSUE TITLE
unused_optional_binding false positive

### ISSUE THREAD

**Issue by keith** [realm/SwiftLint#1210]
```swift
guard let a = b else {
    return
}

private typealias Foo = (_ completion: @escaping (_ bar: Baz) -> Void) -> Void
```

// @rakaramos

**Comment by marcelofabri** [comment]
@keith Could you please test against `master`? I wasn't able to reproduce it here.

**Comment by jpsim** [comment]
I also can't reproduce this.

**Comment by keith** [comment]
I was using 0.16.0 so let me try again with master.


--
Keith Smiley

**Comment by keith** [comment]
Ok yep, I can't reproduce this on master either. Closing
